package.json

```diff
@@ -43,15 +43,15 @@
     "@commitlint/config-conventional": "^18.6.2",
     "@semantic-release/changelog": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
-    "@storybook/addon-actions": "^7.6.15",
-    "@storybook/addon-essentials": "^7.6.15",
-    "@storybook/addon-interactions": "^7.6.15",
-    "@storybook/addon-links": "^7.6.15",
-    "@storybook/addon-themes": "^7.6.15",
-    "@storybook/blocks": "^7.6.15",
-    "@storybook/preview-api": "^7.6.15",
-    "@storybook/react": "^7.6.15",
-    "@storybook/react-vite": "^7.6.15",
+    "@storybook/addon-actions": "^8.0.4",
```

Comment:
upgraded storybook to v8, fixes import problems and slow build speed experienced in v7. Also no longer ignores our vite config.